<launch>

<!-- Initial Paramteres -->
	<arg name="max_lin" default="1."/>
	<arg name="max_ang" default="1."/>


<!-- joy node -->
	<node respawn="true"
		pkg="joy"
		type="joy_node" 
		name="robot_joy" >
			
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.001" />
		<param name="autorepeat_rate" type="double" value="0" />
		<remap from="joy" to="/soybot/joy"/> 
  	</node>


<!-- Teleoperete node -->
	<node name="teleop_joy" 
    	  pkg="soybot_simulator" 
		  type="teleop_main.py"
		  args="$(arg max_lin) $(arg max_ang)">
		  <!--output='screen'-->
		    
		<remap from="cmd_vel" to="/soybot/cmd_vel"/>
		<remap from="joy" to="/soybot/joy"/>
		<remap from="restart" to="/visual_servoing/reset"/>
		<remap from="pub_visual" to="/visual_servoing/start"/>

	</node>

</launch>
